hack/pin-bundle-images.sh

```diff
@@ -88,4 +88,8 @@ done
 if [ -z "$DOCKERFILE" ]; then
     # pin rabbit to sha256 for the e2d413c102bc73d4b5e186d1d1b1f9bf728701e1 tag (v2.6.0)
     echo -n ",quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:261306a5a39a379079a7f30491491ce1bd3ae3e21d9610af9734e4e56d32358d"
+
+    REPO_CURL_URL="https://quay.io/api/v1/repository/openstack-k8s-operators"
+    SHA=$(curl -s ${REPO_CURL_URL}/test-operator-bundle/tag/?onlyActiveTags=true?limit=100 | jq -r .tags[].name | sort -u | head -n 1)
+    echo -n ",quay.io/openstack-k8s-operators/test-operator-bundle:${SHA}"
```

Comment:
rather than this we could add a mute go.mod entry by importing the scheme for test-operator into main.go. And thereby have a promotion for this operator.